tasks/main.yml

```diff
@@ -206,7 +206,9 @@
   file:
     path: /etc/systemd/system/keepalived.service.d/
     state: directory
-  when: ansible_service_mgr == 'systemd'
+  when:
+    - ansible_service_mgr == 'systemd'
+    - keepalived_systemd_overrides
```

Comment:
could you add "| bool" , so that keepalived_systemd_overrides gets validated, please?